### PR TITLE
Implement navigation bar

### PR DESCRIPTION
**Pull request by YoonYeoHwan**
Issue #10

### CODE REVIEW

File: web/src/App.js [hunk 1]
```diff
@@ -1,19 +1,23 @@
...
+import {ApolloProvider} from 'react-apollo';
+import NavBar from './components/navigation/navbar';
 
 const client = new ApolloClient({
-    uri: "http://localhost:4000"
+  uri: 'http://localhost:4000',
 });
 
 function App() {
+  // TODO : add a state to check if the user is logged in now
```
Comment (shberry) [inline web/src/App.js]: TODO(someone)

Comment (shberry) [reply web/src/App.js]: Add (first letter of the first word of a sentence should be capitalized)

Comment (shberry) [reply web/src/App.js]: Comma at the end

File: web/src/App.js [hunk 2]
```diff
@@ -1,19 +1,23 @@
...
   return (
       <div>
+        <NavBar/>
+        {/* TODO : set this to sign out when the user is logged in */}
         Main
```
Comment (shberry) [inline web/src/App.js]: Looks like this need extra indentation

File: web/src/App.js [hunk 3]
```diff
@@ -1,19 +1,23 @@
...
-    uri: "http://localhost:4000"
+  uri: 'http://localhost:4000',
 });
 
 function App() {
+  // TODO : add a state to check if the user is logged in now
   return (
       <div>
+        <NavBar/>
+        {/* TODO : set this to sign out when the user is logged in */}
```
Comment (shberry) [inline web/src/App.js]: TODO(someone). Same comment as above TODO.

File: web/src/components/navigation/navbarmobile.js [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+  };
+
+  const sideList = side => (
+      <div
+          className={classes.list}
+          role="presentation"
+          onClick={toggleDrawer(side, false)}
+          onKeyDown={toggleDrawer(side, false)}
+      >
+        <List>
```
Comment (shberry) [inline web/src/components/navigation/navbarmobile.js]: Weird indentation

File: web/src/components/navigation/navbarmobile.js [hunk 5]
```diff
@@ -0,0 +1,125 @@
...
+          <a href="/" style={{padding: 20}}>
+            <img src={Logo} width="130" height="18"/>
+          </a>
+        </List>
+        <Divider/>
+        <List>
+          <Grid
```
Comment (shberry) [inline web/src/components/navigation/navbarmobile.js]: maybe you can put 'container' in this line?

File: web/src/components/navigation/navbarmobile.js [hunk 6]
```diff
@@ -0,0 +1,125 @@
...
+              container
+              direction="column">
+            <a href="/" className={classes.listItem}>
+              <Typography variant="button">
+                Events
+              </Typography>
+            </a>
+            {/* TODO : need to implement About page. Now just linked to landing page */}
```
Comment (shberry) [inline web/src/components/navigation/navbarmobile.js]: TODO(someone)

File: web/src/pages/signin.js [hunk 7]
```diff
@@ -0,0 +1,16 @@
+import React, {Component} from 'react';
+import PropTypes from 'prop-types';
+
+class Signin extends Component {
+  render() {
+    return (
+        <div>
+          Here is the sign in page
```
Comment (shberry) [inline web/src/pages/signin.js]: Is this the finalized wording? If not, add a todo to update this.

File: web/src/components/navigation/navbarmobile.js [hunk 8]
```diff
@@ -0,0 +1,125 @@
...
+} from '@material-ui/core';
+
+const useStyles = makeStyles({
+  list: {
+    width: 250,
+  },
+  listItem: {
+    margin: 10,
+    marginLeft: 20,
+    color: '#000000',
```
Comment (shberry) [inline web/src/components/navigation/navbarmobile.js]: #000 does this work? 

Or can you check whether there's a predefined variable for this?

File: web/src/components/navigation/navbarmobile.js [hunk 9]
```diff
@@ -0,0 +1,125 @@
...
+  List,
+  makeStyles,
+  Divider,
+  Button,
+  Typography, Tab, Tabs,
+} from '@material-ui/core';
+
+const useStyles = makeStyles({
+  list: {
+    width: 250,
```
Comment (shberry) [inline web/src/components/navigation/navbarmobile.js]: Is it possible to combine these constants in another file (e.g. constants.js or something like this)? I can see same variables being used in multiple files.

File: web/src/components/navigation/navbarmobile.js (hunk 8 above)
Comment (YoonYeoHwan) [reply web/src/components/navigation/navbarmobile.js]: #000 works. But changed it to 'black'.

File: web/src/App.js (hunk 2 above)
Comment (YoonYeoHwan) [reply web/src/App.js]: for NavBar? or TODO? I don't understand exactly.

File: web/src/components/navigation/navbarmobile.js (hunk 9 above)
Comment (YoonYeoHwan) [reply web/src/components/navigation/navbarmobile.js]: I made a constants.js file and separated an useStyles.
I can't find files that used same variables. Could you tell me which file is?

File: web/src/App.js [hunk 10]
```diff
@@ -1,19 +1,24 @@
...
+        {/* TODO(Myoung-heeSeo) : Set this to sign out when the user is logged in. */}
         Main
-        <Landing client={client} />
+        <Landing client={client}/>
       </div>
   );
 }
 
-export default App;
+export default App;
```
Comment (shberry) [inline web/src/App.js]: add new line at the end of the file?

File: web/src/components/navigation/navbarpc.js [hunk 11]
```diff
@@ -0,0 +1,60 @@
...
+function NavBarPC() {
+  return (
+      <BrowserRouter>
+        <div>
+          <Route
+              path="/"
+              render={({location}) => (
+                  <>
+                    <AppBar color="default">
+                      <Grid container direction="row" justify="space-between"
```
Comment (shberry) [inline web/src/components/navigation/navbarpc.js]: Usually attribute are listed in lexicographical order (so it's easier to read and keep track)

File: web/src/pages/signin.js [hunk 12]
```diff
@@ -0,0 +1,18 @@
+import React, {Component} from 'react';
+import PropTypes from 'prop-types';
+import {Tabs} from '@material-ui/core';
+
+class Signin extends Component {
+  render() {
+    return (
+        <div>
+          {/* TODO(Myoung-heeSeo) : Need to implement About page. Now just linked to landing page. */}
+          Here is the sign in page
```
Comment (shberry) [inline web/src/pages/signin.js]: Is this the final wording? If not, add a TODO to update this text

File: web/src/pages/signup.js [hunk 13]
```diff
@@ -5,7 +5,7 @@ class Signup extends Component {
   render() {
     return (
         <div>
-
+          Here is the sign up page
```
Comment (shberry) [inline web/src/pages/signup.js]: Same here

File: web/src/components/navigation/navbarmobile.js [hunk 14]
```diff
@@ -0,0 +1,111 @@
...
+        </List>
+      </div>
+  );
+
+  return (
+      <div>
+        <BrowserRouter>
+          <div className="Navbar">
+            <Route
+                path="/"
```
Comment (shberry) [inline web/src/components/navigation/navbarmobile.js]: this can be moved to above line

File: web/src/components/navigation/navbarpc.js [hunk 15]
```diff
@@ -0,0 +1,61 @@
...
+                      </a>
+                      <a href="/signin">
+                        <Typography variant="h6" style={{textTransform: 'none', color: 'black', marginLeft: 20, marginRight: 30}}>
+                          Sign in
+                        </Typography>
+                      </a>
+                    </Grid>
+                  </Grid>
+                </Grid>
+
```
Comment (shberry) [inline web/src/components/navigation/navbarpc.js]: remove extra line

File: web/src/pages/about.js [hunk 16]
```diff
@@ -0,0 +1,15 @@
+import React, { Component } from "react";
+
+class About extends Component {
+  render() {
+    return (
+        // TODO(YoonYeoHwan): Implement about page.
+        <div>
+           {/* TODO(YoonYeoHwan): Delete text after implement this page. */}
```
Comment (shberry) [inline web/src/pages/about.js]: Weird indentation

File: web/src/components/navigation/navbarmobile.js [hunk 17]
```diff
@@ -0,0 +1,110 @@
...
+
+function NavBarMobile() {
+  const classes = useStyles();
+  const [state, setState] = React.useState({
+    left: false,
+  });
+
+  const toggleDrawer = (side, open) => event => {
+    if (event.type == 'keydown' &&
+      (event.key === 'Tab' || event.key === 'Shift')) {
```
Comment (ethkim) [inline web/src/components/navigation/navbarmobile.js]: weird indentation

**Pull request merged**